Gemfile

```diff
@@ -23,6 +23,7 @@ gem 'jbuilder', '~> 2.0'
 # bundle exec rake doc:rails generates the API under doc/api.
 gem 'sdoc', '~> 0.4.0', group: :doc
 
+gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
```

Comment:
This gem should only be necessary for Windows machines (some background here http://stackoverflow.com/questions/23022258/tzinfodatasourcenotfound-error-starting-rails-v4-1-0-server-on-windows) I don't actually install this gem on my machine because I have a mac so it may have dependencies that I'm missing. I expect them to be automatically pulled into `Gemfile.lock` the first time someone with a windows machine runs `bundle install`.